packages/uni-virtualizer/src/lib/VirtualRepeater.ts

```diff
@@ -0,0 +1,631 @@
+import {ItemBox, Margins} from './layouts/Layout'
```

Comment:
CamelCase file name kinda unusual in our repos, not a big deal but we usually use snake_case.

Reply:
We usually do `kebab-case` in our external repos.

Also, add extensions.